diff --git a/db/src/schema.rs b/db/src/schema.rs
index 0d587b3..4c56438 100644
--- a/db/src/schema.rs
+++ b/db/src/schema.rs
@@ -43,13 +43,13 @@ diesel::table! {
         time_base -> Nullable<Int4>,
         time_increment -> Nullable<Int4>,
         last_interaction -> Nullable<Timestamptz>,
         black_time_left -> Nullable<Int8>,
         white_time_left -> Nullable<Int8>,
         speed -> Text,
-        hashes -> Array<Nullable<Int8>>,
+        hashes -> Array<Int8>,
     }
 }
 
 diesel::table! {
     games_users (game_id, user_id) {
         game_id -> Uuid,
@@ -86,13 +86,7 @@ diesel::table! {
 }
 
 diesel::joinable!(games_users -> games (game_id));
 diesel::joinable!(games_users -> users (user_id));
 diesel::joinable!(ratings -> users (user_uid));
 
-diesel::allow_tables_to_appear_in_same_query!(
-    challenges,
-    games,
-    games_users,
-    ratings,
-    users,
-);
+diesel::allow_tables_to_appear_in_same_query!(challenges, games, games_users, ratings, users,);
